services/web-app/components/feature-card/feature-card.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { AspectRatio, Column, Grid } from '@carbon/react'
+import { ArrowRight } from '@carbon/react/icons'
+import Link from 'next/link'
+import PropTypes from 'prop-types'
+
+import { mediaQueries, useMatchMedia } from '@/utils/use-match-media'
+
+import styles from './feature-card.module.scss'
+
+export const FeatureCard = ({ href, children, description, title }) => {
+  const isMd = useMatchMedia(mediaQueries.md)
+  const isLg = useMatchMedia(mediaQueries.lg)
+
+  const imageAspectRatio = () => {
+    if (isLg) return ''
+    if (isMd) return '16x9'
+    return '1x1'
+  }
+
+  const tileAspectRatio = () => {
+    if (isMd) return '16x9'
+    return '2x1'
+  }
+  const renderElement = () => (
+    <Grid>
+      <Column sm={4} md={8} lg={12}>
+        <Link href={href}>
+          <a className={styles.featureCard}>
+            <AspectRatio ratio={imageAspectRatio()} className={styles.image}>
+              {children}
+            </AspectRatio>
+            <Grid className={styles.gridTile}>
+              <Column sm={4} md={{ offset: 4, span: 4 }} lg={{ offset: 8, span: 4 }}>
+                <AspectRatio ratio={tileAspectRatio()} className={styles.tile}>
+                  <h5 className={styles.title}>{title}</h5>
+                  <p className={styles.description}>{description}</p>
+                  <ArrowRight className={styles.arrowRight} size={20} />
+                </AspectRatio>
+              </Column>
+            </Grid>
+          </a>
+        </Link>
+      </Column>
+    </Grid>
+  )
+
+  return renderElement()
```

Comment:
minor: It looks like the `<Grid>` could be returned directly here.

Reply:
> like the `<Grid>` could be returned direc

Hey Joe! what do you mean by that exactly?

Reply:
Ah, sorry I just meant that you have:
```js
const renderElement = () => (<Grid>...</Grid>)
return renderElement()
```
and thought that could probably be shortened to just:
```js
return <Grid>...</Grid>
```

Reply:
oohh yes of course